### ISSUE TITLE
Wrong TLS setting in Simple Example of readme.md

### ISSUE THREAD

**Issue by hansratzinger** [PHPMailer/PHPMailer#2715]
When I was updating my application to PHPMailer 6.5 using the *Simple Example* published in readme.md, the SMTP server refused to connect. After hours and hours of trying to connect to outlook.office365.com Microsoft ESMTP MAIL Service I found the bug in the misleading comment of *Simple Example*:

````$mail->SMTPSecure = PHPMailer::ENCRYPTION_SMTPS; //Enable implicit TLS encryption````

But in spite of the comment this was starting SSH. I changed it in my script to

````$mail->SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS; ````

and it worked! Please change the comment in the *Simple Example*!

**Comment by Synchro** [comment]
Uh, no. That's not the only setting involved. The TLS mode will also require you to select the appropriate port. SMTP+STARTTLS (explicit TLS) will typically work on port 25 or 587. SMTPS (implicit TLS) requires port 465. **You cannot mix these modes**. If you're using SMTP+STARTTLS mode, you are *not* using implicit TLS. The configuration in the example is correct.

**Comment by hansratzinger** [comment]
I already used port 587. When connecting to outlook.office365.com Microsoft ESMTP MAIL Service the response using PHPMailer::ENCRYPTION_SMTPS was:
````
2022-06-20 15:52:50 SMTP ERROR: Failed to connect to server: (0)
SMTP Error: Could not connect to SMTP host.
````
With  PHPMailer::ENCRYPTION_STARTTLS the server is connecting:
`````
2022-06-20 15:46:02 CLIENT -> SERVER: STARTTLS
2022-06-20 15:46:02 SERVER -> CLIENT: 220 2.0.0 SMTP server ready
`````
My fault was to look only for TLS and not to understand the difference between implicit and explicit TLS. 
[Docs Smtp.office365.com](https://docs.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/pop3-and-imap4/pop3-and-imap4#settings-users-use-to-set-up-pop3-or-imap4-access-to-their-exchange-online-mailboxes)

In conclusion it would be helpful to add to *Simple Example*
`````
// $mail->SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS;  // Enable explicit TLS encryption
`````

**Comment by Synchro** [comment]
This is also covered in detail in [the docs](https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting#encryption-flavours).
